packages/cli/src/Ldap/helpers.ts

```diff
@@ -202,7 +202,13 @@ export const processUsers = async (
 					providerId: ldapId,
 				});
 				if (authIdentity?.userId) {
-					await transactionManager.update(User, { id: authIdentity?.userId }, { disabled: true });
+					const user = await transactionManager.findOneBy(User, { id: authIdentity.userId });
+
+					if (user) {
+						user.disabled = true;
+						await transactionManager.save(user);
+					}
+
```

Comment:
Can't we just use `await transactionManager.save({id: authIdentity.userId, disabled: true});` ?

Reply:
We could, but that means if the there is no user with the id `userId` it would create a row in the db for it.
That's not how the old code behaved, which was using `update`. `update` is a no-op if nothing matches. `save` is an insert if nothing matches, but an update if something does match.

Reply:
Yeah I thought about that, and it makes sense to be consistent with the approach we follow, so let's keep it like this.